src/main/java/org/opensearch/geospatial/index/mapper/xypoint/XYPoint.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.index.mapper.xypoint;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * Represents a point in a 2-dimensional planar coordinate system with no range limitations
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+public class XYPoint {
+    @Getter
+    private double x;
+    @Getter
+    private double y;
```

Comment:
You can move @Getter to the class level, seems all properties have individual getters

Reply:
ack